Src/Base/AMReX_FabConv.cpp

```diff
@@ -524,7 +524,7 @@ permute_real_word_order (void*       out,
                          const int*  inord, 
                          int         REALSIZE)
 {
-    BL_PROFILE("permute_real_word_order");
+//    BL_PROFILE("permute_real_word_order");
```

Comment:
This is intentional I take it, because these functions are called too many times?

Reply:
These functions are now used by the background thread.  It messes up the profiling tools.  Since they don't take much time anyway, I took them out.